assets/helpers/checkoutFormIsSubmittableActions.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { canContributeWithoutSigningIn } from 'helpers/identityApis';
+import type { State } from '../pages/new-contributions-landing/contributionsLandingReducer';
```

Comment:
we shouldn't have helpers depending on page-specific code

Reply:
since page-specific code often depends in turn on helper functions, this kind of thing will make circular dependencies more likely.

Reply:
Ok, really I just want it to be able to be called by both the UserActions and ContributionsLandingActions. Could I move this to the same level as `ContributionsLandingActions`? What do you think makes sense?

Reply:
ohhhhh I see.... hmmm